basecoin/store/src/types/height.rs

```diff
@@ -11,16 +9,6 @@ pub enum Height {
     Stable(RawHeight), // or equivalently `tendermint::block::Height`
 }
 
-impl Display for Height {
```

Comment:
Curious why this `Display` impl was removed?

Reply:
Hm. I don't exactly remember now. But probably because it is not being used anywhere.

Reply:
Also, it has `Debug` implemented. So if downstream projects want to print it - they can use `{:?}`.